jdk/src/share/native/java/net/net_util.c

```diff
@@ -23,11 +23,41 @@
  * questions.
  */
 
+/*
+ * ===========================================================================
+ * (c) Copyright IBM Corp. 2025, 2025 All Rights Reserved
+ * ===========================================================================
+ */
+
 #include "jni.h"
 #include "jvm.h"
 #include "jni_util.h"
 #include "net_util.h"
 
+/* These definitions required by j9.h are in the OpenJ9 jni.h, but OpenJDK jni.h is used here. */
+struct GCStatus;
+typedef struct GCStatus GCStatus;
+struct JavaVMQuery;
+typedef struct JavaVMQuery JavaVMQuery;
+struct JVMExtensionInterface_;
+typedef const struct JVMExtensionInterface_ *JVMExt;
+
+#define COPY_PROGRESS_INFO_MASK 0
+#if defined(AIX)
+#define AIXPPC
+#define RS6000
+#endif /* defined(AIX) */
+#if defined(MACOSX)
+#define OSX
+#endif /* defined(MACOSX) */
+#if defined(WIN32)
+#define OMR_OS_WINDOWS
+#endif /* defined(WIN32) */
+
+#include "j9.h"
+#include "tracehelp.c"
```

Comment:
I think this sequence should be captured in a new header file (perhaps in openj9) to eliminate the repetition.

Reply:
Created jcltracepoints.h in the OpenJ9 include directory.